plugins/org.jkiss.dbeaver.ext.cubrid/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: DBeaver Cubrid Support
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.cubrid;singleton:=true
+Bundle-Version: 1.0.189.qualifier

```

Comment:
There are different versions in pom.xml (for maven) and manifest (for osgi).
So an mvn build error occurs.
I hope the first version is 1.0.0.

Reply:
and After all PR, please check the results using Maven. (mvn package -Pall-platforms)